lib/twofishes/client.rb

```diff
@@ -1,53 +1,53 @@
-require 'httparty'
-
 module Twofishes
   class Client
     # @see https://github.com/foursquare/twofishes/blob/master/docs/twofishes_requests.md
 
-    include HTTParty
-    # debug_output $stderr # httparty debugging
-
     # Geocodes a given string.
     #
-    # @param [String] query
+    # @param [String] location
+    # @param [Array] list of additional ResponseIncludes constants
     # @return [Twofishes::Result]
     #
     # @example
-    #     Twofishes::Client.geocode('Zurich, Switzerland')
+    #   Twofishes::Client.geocode('Zurich, Switzerland')
     #
-    def self.geocode(location)
-      call_api(query: location)
+    def self.geocode(location, response_includes = [])
+      handle_response do
+        thrift_client.geocode(GeocodeRequest.new(query: location, responseIncludes: response_includes))
+      end
     end
 
     # Reverse geocodes lat/lng.
     #
-    # @param [Float] lat
-    # @param [Float] lng
-    # @return [Twofishes::Result]
-    #
+    # @overload reverse_geocode(lat, lng)
+    #   @param [Float] latitude
+    #   @param [Float] longitude
+    #   @return [Twofishes::Result]
+    # @overload reverse_geocode([lat, lng])
+    #   @param [Array] latitude, longitude pair
+    #   @return [Twofishes::Result]
     # @example
-    #     Twofishes::Client.reverse_geocode(47.3787733, 8.5273363)
+    #   Twofishes::Client.reverse_geocode(47.3787733, 8.5273363)
+    #   Twofishes::Client.reverse_geocode([47.3787733, 8.5273363])
     #
-    def self.reverse_geocode(coordinates)
-      call_api(ll: coordinates.join(','))
-    end
-
-    def self.call_api(params)
+    def self.reverse_geocode(*coordinates)
```

Comment:
Well you had 
```
# @param [Float] lat		+    # @overload reverse_geocode(lat, lng)
# @param [Float] lng
```
in documentation but `Twofishes::Client.reverse_geocode([0, 0])` in readme so I don't know what you wanted.